README.md

```diff
@@ -323,20 +341,125 @@ end
+| `cryptographic_binding_methods_supported` | List of supported DID Methods                      | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.2.3-2.11.2.3)   | Accepted values are `did:web`, `did:jwk`, or `did:dht`                         |
+| `credential_signing_alg_values_supported` | List of supported cryprographic signing algorithms | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.2.3-2.11.2.4)   | Accepted values are `EdDSA` or `ES256K`                                        |
+| `proof_types_supported`                   | Object that describes the supported key proof      | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.2.3-2.11.2.5.1) | MUST be `{"jwt": {"proof_signing_alg_values_supported": ["EdDSA", "ES256K"]}}` |
+
+[Reference](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.2.3-2.11.1)
+
+#### Authorization Server Metadata
+| Field            | Description                                               | Required | References                                                         | Comments                          |
+| :--------------- | :-------------------------------------------------------- | :------- | :----------------------------------------------------------------- | :-------------------------------- |
+| `issuer`         | URL of then Credential Issuer                             | y        | [RFC8414](https://datatracker.ietf.org/doc/html/rfc8414#section-2) | Same value as `credential_issuer` |
+| `token_endpoint` | URL for the [Access Token Request](#access-token-request) | y        | [RFC8414](https://datatracker.ietf.org/doc/html/rfc8414#section-2) |                                   |
+
+[Reference](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-11.3)
+
+[Reference](https://datatracker.ietf.org/doc/html/rfc8414)
+
+### 2. Access Token
+
+#### Access Token Request
+| Field        | Description                                                                       | Required | References                                                                                                                 | Comments                                                       |
+| :----------- | :-------------------------------------------------------------------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------- |
+| `grant_type` |                                                                                   | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-3.5-3)                         | MUST be `urn:ietf:params:oauth:grant-type:pre-authorized_code` |
```

Comment:
Not going to do this one because the text on RFC6749 is, `Value MUST be set to "authorization_code"` which is not the case here

Feel free to propose in a follow-up